CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 85.12.0
+
+This minor release introduces Material Elevation support for ActionSheet and Banner (Beta). It also includes performance improvements for Ink and Ripple.
+
+## New features
+
+Support for Material Elevation expanded to two more components. APIs to support
+overriding the computed value for `mdc_baseElevation` and to respond to
+elevation changes are available. Assigning an `elevationDidChangeBlock` on a
+supported component enables code to be executed whenever the component's
+elevation changes. This may include state-based elevation changes (like
+pressing on a Chip) or when the component's elevation value is changed
+programmatically.
+
```

Comment:
Call out the performance improvements details here as well:

```
## Performance improvements

Changes to how Ink and Ripple create colors will result in memory savings on the order of KBs for apps that have on the order of hundreds of ink/ripple instances in memory. Memory fragmentation, cache performance, malloc contention will also be reduced slightly.
```

Reply:
Sorry, didn't see your review comments before I started on the merging work.

Reply:
Please include it in the release notes on GitHub and in the internal email then.

Reply:
Let's talk offline. That would violate our release process.